packages/react/package.json

```diff
@@ -1,43 +1,85 @@
 {
-  "name": "@macaron-css/react",
-  "version": "1.5.3",
+  "$schema": "https://json.schemastore.org/package.json",
+  "name": "@mincho-js/react",
+  "description": "React utilities for Mincho",
+  "version": "0.0.3",
+  "sideEffects": false,
+  "type": "module",
   "license": "MIT",
-  "main": "dist/index.js",
-  "module": "dist/index.mjs",
-  "types": "dist/index.d.ts",
   "repository": {
     "type": "git",
-    "url": "https://github.com/mokshit06/macaron.git",
+    "url": "git+https://github.com/mincho-js/mincho.git",
     "directory": "packages/react"
   },
+  "keywords": [
+    "react",
+    "css",
+    "style",
+    "typescript",
+    "vanilla-extract"
+  ],
+  "bugs": {
+    "url": "https://github.com/mincho-js/mincho/issues"
+  },
+  "homepage": "https://github.com/mincho-js/mincho/tree/main/packages/react#readme",
+  "typings": "./dist/esm/index.d.ts",
+  "main": "./dist/cjs/index.cjs",
+  "module": "./dist/esm/index.mjs",
   "exports": {
     ".": {
-      "import": "./dist/index.mjs",
-      "require": "./dist/index.js"
+      "import": {
+        "types": "./dist/esm/index.d.ts",
+        "default": "./dist/esm/index.mjs"
+      },
+      "require": {
+        "types": "./dist/cjs/index.d.cts",
+        "default": "./dist/cjs/index.cjs"
+      }
     },
-    "./dist/*": "./dist/*",
     "./runtime": {
-      "import": "./dist/runtime.mjs",
-      "require": "./dist/runtime.js"
+      "import": {
+        "types": "./dist/esm/runtime.d.ts",
+        "default": "./dist/esm/runtime.mjs"
+      },
+      "require": {
+        "types": "./dist/cjs/runtime.d.cts",
+        "default": "./dist/cjs/runtime.cjs"
+      }
     }
   },
-  "dependencies": {
-    "@macaron-css/core": "1.5.2"
-  },
   "files": [
     "dist",
     "src"
   ],
-  "peerDependencies": {
-    "@vanilla-extract/recipes": "^0.2.5",
-    "react": ">=17",
-    "react-dom": ">=17"
+  "scripts": {
+    "clean": "yarn g:clean",
+    "build": "yarn g:build",
+    "build:watch": "yarn g:build-watch",
+    "test": "yarn g:test",
+    "test:watch": "yarn g:test-watch",
+    "coverage": "yarn g:coverage",
+    "lint": "yarn g:lint",
+    "fix": "yarn g:fix",
+    "check": "yarn g:check"
+  },
+  "prettier": "prettier-config-custom",
+  "dependencies": {
+    "@mincho-js/css": "workspace:*"
   },
   "devDependencies": {
-    "@types/react": "^18.0.12",
-    "@types/react-dom": "^18.0.6",
+    "@types/react": "^19.0.8",
+    "@types/react-dom": "^19.0.3",
+    "@vanilla-extract/recipes": "^0.2.5",
+    "eslint-config-custom": "workspace:^",
+    "prettier-config-custom": "workspace:^",
+    "react": "^19.0.0",
+    "react-dom": "^19.0.0",
+    "tsconfig-custom": "workspace:^",
+    "vite-config-custom": "workspace:^"
+  },
+  "peerDependencies": {
     "@vanilla-extract/recipes": "^0.2.5",
-    "react": "^18.1.0",
-    "react-dom": "^18.2.0"
+    "react": ">=19",
+    "react-dom": ">=19"
```

Comment:
I don't want to heavily restrict the version of React that users install. It might be better to stick with the original version.